demos/dashy/index.html

```diff
@@ -7,11 +7,11 @@
     <link href="style/index.css" rel="stylesheet">
 
     <script>
-        var mode = "dev";  // "dev" | "rollup" | "webpack" |  "unpkg"
+        var mode = "dist";  // "dev" | "rollup" | "webpack" |  "unpkg" | "dist"
```

Comment:
Maybe we should have separate html files for these different modes?

Reply:
Not worth the effort I suspect